packages/video_player/video_player_platform_interface/pigeons/messages.dart

```diff
@@ -55,6 +57,7 @@ abstract class VideoPlayerApi {
   void seekTo(PositionMessage msg);
   void pause(TextureMessage msg);
   void setMixWithOthers(MixWithOthersMessage msg);
+  void setupAudioSession(AudioSessionMessage msg);
```

Comment:
You don't need to change the default implementation; it's for compatibility with existing third-party implementations that haven't updated to implementing the platform interface directly. Legacy implementations won't be looking for this call.

(In fact, adding this would make your PR do the wrong thing with legacy implementations, because they would throw during init due to this not being implemented on the native side; the behavior you want is to no-op.)